### DISCUSSION TITLE
Table popup editing cancel button not working

### DISCUSSION THREAD

**Discussion by botura** [quasarframework/quasar#10213] [General - Components / Directives / etc] [answered]
I might be missing something here, even if I click the cancel button, the value is updated (see calories column). Actually the model is being updated instantly.

https://codepen.io/botura/pen/ZEKrmBE?editors=1011

I'm almost sure Quasar v1 doesn't have this behavior.

Even [v2 doc](https://quasar.dev/vue-components/table#popup-editing) doesn't work as well as [v1 doc](https://v1.quasar.dev/vue-components/table#popup-editing)

**Comment by metalsadman** [comment, marked as answer]
@botura the examples needs updating as it's still using the old way. See the ones in the QPopupEdit section https://quasar.dev/vue-components/popup-edit#example--persistent-edit-and-with-buttons these are the correct way to implement in Quasar v2. there's also a warning there https://quasar.dev/vue-components/popup-edit#usage, so key is use QPopupEdit's default slot to forward its scoped properties `q-popup-edit ... v-slot="scope"` see https://quasar.dev/vue-components/popup-edit#the-default-slot.

Also mentioned in the upgrade guide https://quasar.dev/start/upgrade-guide#qpopupedit.

**Comment by botura** [comment]
Thank you for your help @metalsadman
